server/data/mappers/prisonerSummaryMapper.ts

```diff
@@ -0,0 +1,21 @@
+import type { Prisoner } from 'prisonRegisterApiClient'
+import type { PrisonerSummary } from 'viewModels'
+import moment from 'moment/moment'
+
+export default function toPrisonerSummary(prisoner: Prisoner): PrisonerSummary {
```

Comment:
This mapper function is essentially exactly what we previously did in `routerRequestHandlers.ts`